packages/tldraw/src/lib/shapes/draw/toolStates/Drawing.ts

```diff
@@ -168,7 +168,9 @@ export class Drawing extends StateNode {
 		const { z = 0.5 } = this.info.point
 
 		this.isPen = isPen
-		this.isPenOrStylus = isPen || (z > 0 && z < 0.5) || (z > 0.5 && z < 1)
+		// if z === 0 on the initial point, treat this pen as a mouse because it's likely a broken pen
+		// or a broken OS.
+		this.isPenOrStylus = (isPen && z !== 0) || (z > 0 && z < 0.5) || (z > 0.5 && z < 1)
```

Comment:
We'd need to test this across browsers. IIRC Safari (?) reports zero pressure in unusual scenarios, such as at the start of lines when using the Apple Pencil. I vaguely remember removing a zero check.

I actually wouldn't mind a "Input device: auto / mouse / pen" setting, similar to the "mousewheel behavior". Better input controls have been on my list forever.